commands/report_generate.js

```diff
@@ -0,0 +1,83 @@
+const request = require("request")
+const constants = require("./utils/constants.js")
+const process = require("process")
+
+
```

Comment:
NIT: Please remove one empty line from 4-5 line. One empty line is sufficient to generate difference.